packages/components/src/components/contract-card/contract-card-items/contract-card-body.jsx

```diff
@@ -237,27 +209,181 @@ const ContractCardBody = ({
                         {status === 'loss' && <Icon icon='IcLoss' />}
                     </div>
                 </ContractCardItem>
-                <ContractCardItem header={getCardLabels().PURCHASE_PRICE}>
-                    <Money amount={buy_price} currency={currency} />
-                </ContractCardItem>
-                <ContractCardItem header={getCardLabels().POTENTIAL_PAYOUT}>
-                    <Money currency={currency} amount={payout} />
-                </ContractCardItem>
-            </div>
-            <MobileWrapper>
-                <div className='dc-contract-card__status'>
-                    {is_sold ? (
-                        <ResultStatusIcon
+                <ContractCardItem header={getCardLabels().TAKE_PROFIT} className='dc-contract-card__take-profit'>
```

Comment:
Is it possible to call this function one time and assign everything we need only once?
Like this:
`const { TAKE_PROFIT, POTENTIAL_PAYOUT, ...} = getCardLabels()`

Reply:
@NikitK-deriv unfortunately, it doesn't seem feasible to me because there is too much of everything we need from getCardLabels() that is used inside this file in various scopes. So we'll have to destructure it in each of the 3 scopes again and again: MultiplierCardBody, AccumulatorCardBody, ContractCardBody. So IMHO it's totally fine to use it the way it's been used for a very long time. Do you agree?